src/AppOriginal.js

```diff
@@ -0,0 +1,171 @@
+import React from "react";
+import "./App.css";
+import { makeShuffledDeck } from "./utils.js";
+
+import { Container, Row, Col } from "react-bootstrap";
+//
+
+// REFER TO IMPORT PNG IN REACT IN CHATGPT TO SOLVE IMAGES ISSUE
+
+class App extends React.Component {
+  constructor(props) {
+    // Always call super with props in constructor to initialise parent class
+    super(props);
+    this.state = {
+      // Set default value of card deck to new shuffled deck
+      cardDeck: makeShuffledDeck(),
+      // currCards holds the cards from the current round
+      currCards: [],
+      winner: null, // Add a winner state variable
+      player1Score: 0,
+      player2Score: 0,
+    };
+
+    // Define newCurrCards as a class property so it can be accessed outside dealCards()
+    this.newCurrCards = [];
+    this.player1 = {};
+    this.player2 = {};
+  }
+
+  dealCards = () => {
+    if (this.state.cardDeck.length < 2) {
+      // Check if there are enough cards in the deck
+      return;
```

Comment:
I think this check is good, but you are not handling it however. A simple return would leave the user with a blank screen. Maybe do something else, like making a new deck, rendering some text inform the user about what happens etc.